src/Concordium/Client/Output.hs

```diff
@@ -591,16 +619,17 @@ showOutcomeResult verbose = \case
 -- of text that they confirmed manually.
 -- The verbose version is used by 'transaction status' and the non-trivial cases of the above
 -- where there are multiple distinct outcomes.
-showEvent :: Verbose -> Types.Event -> Maybe String
-showEvent verbose = \case
+showEvent :: Verbose -> Maybe SchemaType -> Types.Event -> Maybe String
+showEvent verbose stM = \case
   Types.ModuleDeployed ref->
     verboseOrNothing $ printf "module '%s' deployed" (show ref)
   Types.ContractInitialized{..} ->
-    verboseOrNothing $ printf "initialized contract '%s' using init function '%s' from module '%s' with %s"
-                              (show ecAddress) (show ecInitName) (show ecRef) (showCcd ecAmount)
+    verboseOrNothing $ [i|initialized contract '#{ecAddress}' using init function '#{ecInitName}' from module '#{ecRef}' |]
+                    <> [i|with #{ecAmount}\n#{showLoggedEvents ecEvents}|]
```

Comment:
Please use `showCcd` and **not** the Show instance of Amount. The former renders it in a uniform way everywhere in the client. The latter is the simple show instance used for debugging.

Also generally do not do refactoring unless necessary. It is very easy to make simple mistakes like this.

Reply:
I'm to blame for the small refactoring, as I suggested it.